File: app/src/main/java/com/fanwe/log/AppLogger.java
package com.fanwe.log;

import android.util.Log;

import com.fanwe.lib.log.FLogger;

public class AppLogger extends FLogger
{
    public static final String TAG = AppLogger.class.getSimpleName();

    @Override
    protected void onCreate()
    {
        openLogFile(50, App.getInstance());
        Log.i(TAG, "onCreate:" + this);
    }
}

File: app/src/main/java/com/sd/demo/log/AppLogger.kt
package com.sd.demo.log

import com.sd.lib.log.FLogger
import com.sd.lib.log.fLogInfo

class AppLogger : FLogger() {
    override fun onCreate() {
        // 开启日志文件，限制最大50MB
        openLogFile(50)
        fLogInfo { "$this onCreate" }
    }
}